src/power.cpp

```diff
@@ -1,10 +1,14 @@
 #include <power.h>
-#include <GUIUtils.hpp>
 
 void prepairShutdown() {
 #ifdef TTGO_TDISPLAY
     digitalWrite(ADC_EN, LOW);
     delay(10);
+    rtc_gpio_init(GPIO_NUM_14);
+    rtc_gpio_set_direction(GPIO_NUM_14, RTC_GPIO_MODE_OUTPUT_ONLY);
+	rtc_gpio_set_level(GPIO_NUM_14, 1);
+    delay(500); 
+ 
 #else
    gui.PowerSave(); 
 #endif
```

Comment:
Maybe is missing the commit of this method?

Reply:
You mean these lines of Guiutils.cpp?
72     void GUIUtils::PowerSave() {
73           u8g2.setPowerSave(1);
74     }

There is all defined, it is a function of the U8g2 library.

Reply:
ok, I didn't see that. It's ok. This class the naming of methods are with non capitalized, change PowerSave to setPowerSave like u8g2 library please